src/webview/index.html

```diff
@@ -19,7 +19,7 @@
       padding: 18px;
       padding-bottom: 22px;
       border-radius: 5px;
-      box-shadow: rgba(0, 0, 0, 0.55) 0px 20px 68px;
+      box-shadow: 0px 5px 20px rgba(0, 0, 0, 0.3);
```

Comment:
Wouldn't this be better with a material design like elevation?
`box-shadow: 0 19px 38px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22);`